bin/e2e-tests-setup.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * File can be run using `wp eval-file` command:
+ * `wp eval-file e2e-tests-setup.php`
+ *
+ * Steps:
+ * 1. Create user "E2ETest Faculty" (admin role).
+ * 2. Create user “E2ETest Student 1” (post author).
+ * 3. Create user “E2ETest Student 2" (author).
+ * 4. Clone course with new site `e2e-testing`.
+ * 5. Make sure “WP Grade Comment” plugin is active.
+ * 5. Set permalink to use post name.
+ * 6. Create post “Test Grade Comment”, author “E2ETest Student 1”.
+ * 7. Create comments.
+ */
+namespace OpenLab\E2ETests\Setup;
+
+use WP_CLI;
+use BP_Groups_Group;
+
+function create_user( $username ) {
+	$exists = \username_exists( $username );
+	if ( $exists ) {
+		WP_CLI::log( "User exists. ID: {$exists}" );
+		return $exists;
+	}
+
+	$password = \wp_generate_password( 24 );
```

Comment:
This means we can never log in as this user, and that we need to have a separate, manually-created account to run authenticated tests. Is that right? Perhaps we can add a `$password = null` parameter here, and use this method to create *all* necessary users.

Reply:
Do you mean to disable password logging?

I agree, with the teardown script, generated passwords, doesn't make much sense. This also will require some updates grade comments tests.

Reply:
> Do you mean to disable password logging?

I mean: the `login()` script relies on a `wp_user` variable that must be set in the config file, using credentials that are created outside of the Cypress flow. https://github.com/livinglab/openlab-e2e-tests/blob/master/cypress/support/commands.js#L7 It seems to me that we could simplify the initialization routine by having the `setup` script create the `wp_user`. (I don't think it's urgent to make this change, but down the road I imagine we will need more complex situations - like WP Grade Comments - that need multiple logins.)

Reply:
Do you mean to generate `cypress.json` with required user creds on dev server via setup script?

P.S. I've also update `login()` command to support multiple users: https://github.com/livinglab/openlab-e2e-tests/pull/1/commits/8680a55e763df0e24e4bd17849bfcc395a456230

Reply:
No, I mean that the credentials could be hardcoded (or generated internally) rather than included as part of the cypress.json file. These are accounts that are only to be used for the automated testing routine - a real person should never need to know or use the credentials.